{"uuid": "cd2cd780-506f-450f-a112-b5bde263f0cc", "children": ["e2b1ef22-2dd5-4010-9940-53561eb8c879"], "befores": [{"name": "driver", "status": "passed", "start": 1751934743370, "stop": 1751934745283}], "afters": [{"name": "driver::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 938, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\test_招聘审批流程.py\", line 34, in driver\n    driver.quit()\n    ~~~~~~~~~~~^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 224, in quit\n    self.service.stop()\n    ~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 150, in stop\n    self.send_remote_shutdown_command()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 135, in send_remote_shutdown_command\n    if not self.is_connectable():\n           ~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 124, in is_connectable\n    return utils.is_connectable(self.port)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\utils.py\", line 98, in is_connectable\n    socket_ = socket.create_connection((host, port), 1)\n  File \"C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.1520.0_x64__qbz5n2kfra8p0\\Lib\\socket.py\", line 856, in create_connection\n    exceptions.clear()  # raise only the last error\n    ~~~~~~~~~~~~~~~~^^\n"}, "start": 1751934760409, "stop": 1751934763033}], "start": 1751934743370, "stop": 1751934763033}